sources/bloodhound/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "bloodhound"
+version = "0.1.0"
+authors = ["Sean McGinnis <stmcg@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[dependencies]
+chrono = "0.4"
```

```suggestion
chrono = { version = "0.4", default-features = false, features = ["std", "clock"] }
```
You can avoid pulling in the old version of `time` by only calling the `chrono` features you need.

---

Tried it out and looks like all we really need is `clock`. Nice clean up of transient dependencies - great suggestion!